### ISSUE TITLE
ReFL Training Performance

### ISSUE THREAD

**Issue by XiaominLi1997** [THUDM/ImageReward#95]
The performance of my ReFL model after training on refl_data.json(https://github.com/THUDM/ImageReward/blob/main/data/refl_data.json) is significantly worse than the untrained SD1.4 model. The results are far from satisfactory, and I'm not sure what might be causing this issue. 

Training Settings:
GPUs: 2 * A100 GPU
--train_batch_size: 8
--gradient_accumulation_steps: 4
--num_train_epochs: 100
--learning_rate: 1e-5

**Comment by xujz18** [comment]
<img width="747" alt="image" src="https://github.com/user-attachments/assets/7d25e79d-00b9-4224-af45-6b6621052b0c">

> In practice, to avoid rapid overfitting and stabilize the fine-tuning, we re-weight ReFL loss and regularize with pre-training loss.

Our code is only used to display the ReFL loss, you will need to add the pre-training loss according to your own settings.

**Comment by XiaominLi1997** [comment]
> <img alt="image" width="747" src="https://private-user-images.githubusercontent.com/52482160/363453985-7d25e79d-00b9-4224-af45-6b6621052b0c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjUxNzEwOTMsIm5iZiI6MTcyNTE3MDc5MywicGF0aCI6Ii81MjQ4MjE2MC8zNjM0NTM5ODUtN2QyNWU3OWQtMDBiOS00MjI0LWFmNDUtNmI2NjIxMDUyYjBjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTAxVDA2MDYzM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTkzNzliMDk0MTk1ZTVmNWFhYmU5ZDI2Yjg2NDNhODZhYTMxZjllYjYxN2JjY2Y2ZTk4N2IxZGE2ZTMwMWIwYmEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.-IL_OJg1vCwzvtpIwmAL-2lmyiXnYbirXV-eDda7umU">
> > In practice, to avoid rapid overfitting and stabilize the fine-tuning, we re-weight ReFL loss and regularize with pre-training loss.
> 
> Our code is only used to display the ReFL loss, you will need to add the pre-training loss according to your own settings.
Thx for your reply.

In this answer (https://github.com/THUDM/ImageReward/issues/24#issuecomment-1597110335), you mentioned that 'it is simpler to use ReFL alone directly and to achieve decent results.' According to your statement, using only the ReFL loss should yield reasonably good results, but I am unable to achieve that. It seems that the loss has already converged.
![image](https://github.com/user-attachments/assets/e58e5ba1-e771-448e-a1eb-99b2250790ed)

Additionally, the paper mentions: 'the pre-training dataset is from a 625k subset of LAION-5B [50] selected by aesthetic score.' I wonder if you plan to release this part of the dataset.

Thanks again.
